pkg/blobfuse-proxy/debpackage/DEBIAN/control

```diff
@@ -1,7 +0,0 @@
-Package: blobfuse-proxy
```

Comment:
this is not needed any more?

Reply:
Yep, I install and start blobfuse-proxy by using binary file directly.